rdt/transformers/categorical.py

```diff
@@ -258,6 +258,35 @@ def _reverse_transform(self, data):
         return self._reverse_transform_by_row(data)
 
 
+class CategoricalFuzzyTransformer(CategoricalTransformer):
+    """Transformer for categorical data.
+
+    This transformer computes a float representative for each one of the categories
```

Comment:
I would reorganize this description a bit to separate the fit and the transform more clearly, starting by saying how the transformer computes the category representative, then mentioning that during transform the categories are replaced by their representative with some gaussian noise, and finish on the reverse transform explanation that we already have.